src/entries/popup/components/Autocomplete/index.tsx

```diff
@@ -0,0 +1,155 @@
+              onBlur={() => setTimeout(onBlur, 200)}
+              onFocus={onFocus}
+              ref={ref}
+            />
+          </Box>
+          {open && (
+            <Box
+              borderColor={'accent'}
+              background="surfacePrimaryElevated"
+              style={{
+                position: 'absolute',
+                zIndex: 999999,
+                width: '100%',
+                boxShadow: '2px 12px 12px rgba(0, 0, 0, 0.65)',
+                borderRadius: defaultBorderRadius,
+              }}
+            >
+              <Command.List>
+                {Object.keys(data).map((key) => {
+                  return (
+                    <Command.Group
+                      heading={key}
+                      key={key}
+                      style={{
+                        color:
+                          semanticColorVars.foregroundColors.labelSecondary,
+                        fontSize: '12px',
+                        padding: '20px 18px',
+                      }}
+                    >
+                      <Box paddingTop="12px">
+                        {data[key].map((item: { name: string }) => (
+                          <Box key={`${key}_${item.name}`}>
+                            <Command.Item
+                              style={{
+                                padding: '8px 12px',
+                                fontSize: '14px',
+                                lineHeight: '20px',
+                                borderRadius: '8px',
+                                outline: 'none',
+                                backgroundColor:
+                                  semanticColorVars.backgroundColors
+                                    .surfacePrimaryElevated,
+                                color: semanticColorVars.foregroundColors.label,
+                              }}
+                              onSelect={() => onSelect(item.name)}
+                              value={item.name}
+                              onMouseOver={(e) => {
+                                e.currentTarget.style.backgroundColor =
+                                  semanticColorVars.backgroundColors.surfaceSecondaryElevated;
+                              }}
+                              onMouseLeave={(e) => {
+                                e.currentTarget.style.backgroundColor =
+                                  semanticColorVars.backgroundColors.surfacePrimaryElevated;
+                              }}
+                            >
+                              {item.name}
+                            </Command.Item>
+                          </Box>
+                        ))}
+                      </Box>
+                    </Command.Group>
+                  );
+                })}
+              </Command.List>
+            </Box>
+          )}
+        </Command>
+      </Box>
+    );
+  },
+);
+
+Autocomplete.displayName = 'Autocomplete';
```

Comment:
jfyi another option to setting a displayName is using a named function in the forwardRef like
```ts
export const Autocomplete = forwardRef(function Autocomplete(props, ref) {
 ...
})
```

Reply:
Good to know! That's probably cleaner than using display name. We should probably fix it across the entire codebase later on